lib/solaar/ui/window.py

```diff
@@ -593,8 +593,9 @@ def _update_receiver_panel(receiver, panel, buttons, full=False):
 
 	may_pair = receiver.may_unpair and not is_pairing
 	if may_pair and devices_count >= receiver.max_devices:
-		online_devices = tuple(n for n in range(1, receiver.max_devices+1) if n in receiver)
-		may_pair &= len(online_devices) < receiver.max_devices
+		 # doesn't this just compute devices_count?
```

Comment:
Yes. You can remove this comment :grin:.

Reply:
Just remove the comment, i.e., keep the computation?  Or remove the entire if statment?